test/functional/test_runner.py

```diff
@@ -100,7 +100,8 @@ class UnicodeOnWindowsError(ValueError):
     # vv Tests less than 60s vv
 
     # vv Tests less than 30s vv
-    'example_test.py',
+    # 'example_test.py',
```

Comment:
Why comment this one?

Reply:
There is some problem with node start up if I enable both tests. I'll need to look into it some more.